templates/autolabel_tma_cores.ipynb

```diff
@@ -54,7 +54,7 @@
     "\n",
```

Comment:
<p>Four corners, not two</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/creed-helper/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>